### ISSUE TITLE
When starting a survey, a timeout in nuntium leads to a running survey

### ISSUE THREAD

**Issue by nekron** [instedd/ask#868]
It shows a "Oops, something went wrong" but the survey remains in running.

We should rollback the transaction updating the survey, or only update it after that callback reaches nuntium.

If the start fails, the survey should not remain in a running state.

**Comment by asterite** [comment]
@nekron I can't reproduce this. I tried disconnecting WIFI right before launching a survey (locally). I get the "Oops" message, but the survey is not launched. Additionally, the code [first prepares the channels](https://github.com/instedd/ask/blob/6bc6ea7e209b937370b734df1203871c102b5c58/web/controllers/survey_controller.ex#L143) and once that's successful it [sets the survey to running](https://github.com/instedd/ask/blob/6bc6ea7e209b937370b734df1203871c102b5c58/web/controllers/survey_controller.ex#L145-L146), so there's no way to set it to "running" if the channel preparation failed.

I tried it both by getting a timeout from nuntium and from OAuthTokenServer (guisso).
